sql/mob_pool_mods.sql

```diff
@@ -59,6 +59,9 @@ INSERT INTO `mob_pool_mods` VALUES (459,73,25,0); -- (Bloodlapper) Store TP
 INSERT INTO `mob_pool_mods` VALUES (459,430,20,0); -- (Bloodlapper) Quad Attack
 INSERT INTO `mob_pool_mods` VALUES (519,160,-50,0);
 INSERT INTO `mob_pool_mods` VALUES (532,160,-50,0);
+INSERT INTO `mob_pool_mods` VALUES (559,62,-50,0); -- Bugbby -ATTP-
+INSERT INTO `mob_pool_mods` VALUES (559,288,40,100); --DATT
```

Comment:
CI is failing here and the line below, space is required between `--` for comment and text afterwards